core/dbt/parser/models.py

```diff
@@ -288,3 +322,54 @@ def _get_sample_result(
             result = ["00_exact_match"]
 
     return result
+
+
+# returns a list of string codes to be sent as a tracking event
+def _get_stable_sample_result(
+    sample_node: ParsedModelNode,
+    sample_config: ContextConfig,
+    node: ParsedModelNode,
+    config: ContextConfig
+) -> List[str]:
+    result: List[str] = []
+    # look for false positive configs
+    for k in config._config_call_dict:
+        if k not in config._config_call_dict:
+            result += ["82_stable_false_positive_config_value"]
```

Comment:
ah, a different error code! really good. I was way overcomplicating this in my head

Reply:
I try to make things easy for you when I can 🙂